nixos/modules/services/networking/mumble-server.nix

```diff
@@ -3,18 +3,18 @@
 with lib;
 
 let
-  cfg = config.services.murmur;
+  cfg = config.services.mumble-server;
   forking = cfg.logFile != null;
-  configFile = pkgs.writeText "murmurd.ini" ''
-    database=/var/lib/murmur/murmur.sqlite
+  configFile = pkgs.writeText "mumble-server.ini" ''
+    database=/var/lib/mumble-server/mumble-server.sqlite
```

Comment:
Won't that break existing installations?

Reply:
It does. I will let this depend on the `stateVersion`, as @K900 suggested. So people will have some time for migration.